scaletest/createworkspaces/run_test.go

```diff
@@ -185,14 +154,139 @@ func Test_Runner(t *testing.T) {
 		require.Len(t, workspaces.Workspaces, 0)
 	})
 
+	t.Run("NoCleanup", func(t *testing.T) {
```

Comment:
review: this is mostly copy-pasta that only serves to validate that the NoCleanup option is respected.